package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dydxprotocol/v3-client",
-  "version": "1.9.1",
+  "version": "1.10.0",
```

Comment:
it's not a breaking change since the field is optional though

Reply:
Discussed in slack; since this is a new feature we should probably keep as 1.10.0